src/OSPSuite.Core/Chart/PredictedVsObservedChart.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Data;
+using OSPSuite.Core.Domain.Services;
+using OSPSuite.Core.Domain.UnitSystem;
+using OSPSuite.Core.Extensions;
+using OSPSuite.Utility.Collections;
+using OSPSuite.Utility.Extensions;
+
+namespace OSPSuite.Core.Chart
+{
+   public abstract class PredictedVsObservedChart : AnalysisChart
+   {
+      private readonly Cache<float, IEnumerable<DataRepository>> _deviationRepositoryCache = new Cache<float, IEnumerable<DataRepository>>(onMissingKey: x => null);
+      public List<float> DeviationFoldValues { get; } = new List<float>();
+
+      protected PredictedVsObservedChart()
+      {
+         ChartSettings.LegendPosition = LegendPositions.BottomInside;
+      }
+
+      /// <summary>
+      ///    Modifies the Y axes visibility based on X axis dimension. Y axes which do not share a unit with X axis are not shown
+      /// </summary>
+      public void UpdateAxesVisibility()
+      {
+         var visibleAxes = Axes.Where(x => x.Dimension != null && x.Dimension.HasSharedUnitNamesWith(XAxis.Dimension)).ToList();
+         visibleAxes.Each(axis => axis.Visible = true);
+         Axes.Except(visibleAxes).Each(axis => axis.Visible = false);
+      }
+
+      public void AddToDeviationFoldValue(float foldValue)
+      {
+         if (DeviationFoldValues.Contains(foldValue))
+            return;
+
+         DeviationFoldValues.Add(foldValue);
+      }
+
+      public override void UpdatePropertiesFrom(IUpdatable source, ICloneManager cloneManager)
```

Comment:
The missing method transfers the foldValues to the new chart but that introduced an issue where if the foldValue was present in the list then the service assumed the curve was present in the chart, but that's not necessarily the case now